src/engine/bank06.asm

```diff
@@ -424,7 +439,134 @@ Func_006_42b1:
 	ret
 
 Data_006_42bb:
```

Comment:
I think this data is too long to contain the same source file. It might be better to split it from the source?

Reply:
I would keep it there for now just because it will probably make your life easier for as long as you're dealing with it. But whatever you prefer. (At least I wouldn't move it until you know what the data is so you can actually name the file where you'd mvoe it to)

Reply:
Yeah. I agree my life is precious :D I'm gonna organize datum after all the works get done.